### PR TITLE
Shade VU-Meter fix

### PR DESCRIPTION
**Pull request by daschuer**
See commits.

### CODE REVIEW

File: res/skins/Shade/preview_deck.xml [hunk 1]
```diff
@@ -201,20 +201,22 @@
...
                     <Pos>2,2</Pos>
                     <Size>5f,8f</Size>
-                    <Connection>
-                      <ConfigKey>[PreviewDeck1],PeakIndicator</ConfigKey>
-                    </Connection>
-                  </StatusLight>
+                    <Layout>horizontal</Layout>
+                    <Children>
+                      <StatusLight>
+                        <TooltipId>sampler_PeakIndicator</TooltipId>
```
Comment (ronso0) [inline res/skins/Shade/preview_deck.xml:211]:
```suggestion
                        <TooltipId>preview_PeakIndicator</TooltipId>
```

**Review by ronso0 — APPROVED** [review]
LGTM, just the two tooltipIds should be replaced

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
Thank you!

**Pull request merged**